code/game/machinery/telecomms/presets.dm

```diff
@@ -175,6 +175,81 @@
 	freq_listening = list(COLONY_FREQ)
 	faction_shorthand = "colony"
 
+GLOBAL_LIST_EMPTY(all_static_telecomms_towers)
+
+/obj/structure/machinery/telecomms/relay/preset/tower/Initialize()
+	GLOB.all_static_telecomms_towers += src
+	. = ..()
+
+/obj/structure/machinery/telecomms/relay/preset/tower/destroy()
+	GLOB.all_static_telecomms_towers -= src
+	. = ..()
+
+/obj/structure/machinery/telecomms/relay/preset/tower/mapcomms
+	name = "TC-3T static telecommunications tower"
+	desc = "A static heavy-duty TC-3T telecommunications tower. Used to set up subspace communications lines between planetary and extra-planetary locations. Will need to have extra communication frequencies programmed into it by multitool."
+	use_power = 0
+	idle_power_usage = 10000
+	icon = 'icons/obj/structures/machinery/comm_tower3.dmi'
+	icon_state = "static1"
+	toggled = FALSE
+	bound_height = 64
+	bound_width = 64
+	freq_listening = list(COLONY_FREQ)
+
+/obj/structure/machinery/telecomms/relay/preset/tower/mapcomms/attack_hand(mob/user)
+	if(stat & NOPOWER)
+		to_chat(user, SPAN_WARNING("\The [src] makes a small plaintful beep, and nothing happens. It seems to be out of power."))
+		return FALSE
+	toggle_state(user) // just flip dat switch
```

Comment:
_Please_ have a `do_after()` of some length somewhere around here. It is being spammed for several screens' worth of logs at a time.

Reply:
added